.gitattributes

```diff
@@ -1,5 +1,7 @@
+.github/ export-ignore
+Tests/ export-ignore
```

Comment:
FWIW I think we shouldn't be excluding Tests from this - they are only included with install-dev. Not if you run a regular install and that's consistent with how most framework packages work

Reply:
> they are only included with install-dev

No, they become part of the distribution zip package, if not excluded explicitely.
Please have a look at this [discussion at Symfony](https://github.com/symfony/symfony/issues/6605), especially [this comment](https://github.com/symfony/symfony/issues/6605#issuecomment-60306001).

The changes (I'm doing that for all Framework packages) just remove the development related files (including the tests) from distribution packages, i.e., the zip file. You still can pull the complete source, as documented in the README.

Reply:
They become part of the distribution package if you're just running composer install. If you run composer install --no-dev then your not getting the dev composer loads which are tests.

Also if you look at the result of that discussion you'll see https://github.com/symfony/symfony/commit/1f762c77230d675158f7c6b29d6eb2a8250eb728 that the end decision was to include the tests and they have been since symfony 2.1